packages/@react-aria/dnd/src/useDrag.ts

```diff
@@ -341,7 +357,6 @@ export function useDrag(options: DragOptions): DragResult {
       onDragEnd
     },
     dragButtonProps: {
-      ...descriptionProps,
```

Comment:
did we mean to delete the descriptionProps when it isn't disabled?

Reply:
That is the "Press Enter to start dragging" description, so I assume we don't want it while disabled.

Reply:
yes but we would want to announce it otherwise right? cause in the [docs](https://reactspectrum.blob.core.windows.net/reactspectrum/78be891d80b00fe7d7b910a8f54ff4dd5b7e8ce2/docs/react-aria/useDrag.html#drag-button) now, it doesn't announce "press enter to start dragging" even when it isn't disabled

Reply:
Oh good catch, that was accidentally removed. Thanks!